src/utils/validation/validation.ts

```diff
@@ -1338,6 +1351,36 @@ export function validateGroup(groupId: string, state: IRuntimeState, onlyInRowIn
   );
 }
 
+/*
+ * Validates a repeating groups component specific validations
+ * @param node the node to validate
+ * @param language the current language
+ */
+export function validateRepeatingGroup(node, language: ILanguage): IComponentValidations {
```

Comment:
Make sure you coordinate with @bjosttveit if he's currently moving these into the component `def` classes